mixpanel/__init__.py

```diff
@@ -15,7 +15,7 @@
 customize the IO characteristics of their tracking.
 """
 
-VERSION = '3.2.0'
+VERSION = '4.0.1'
```

Comment:
No need to bump the versions in your PRQs (although it looks like we missed the bump here in the 4.0 release!)